src/Symfony/Component/Messenger/Transport/Serialization/SerializerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Serialization;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface SerializerInterface extends DecoderInterface, EncoderInterface
```

Comment:
Not sure we need to keep the decoder and encoder interface.

Reply:
I've removed them 👍